core/src/main/scala/sttp/tapir/Validator.scala

```diff
@@ -236,7 +232,7 @@ object Validator extends ValidatorMagnoliaDerivation with ValidatorEnumMacro {
         case MaxLength(value)          => Some(s"length<=$value")
         case MinSize(value)            => Some(s"size>=$value")
         case MaxSize(value)            => Some(s"size<=$value")
-        case Custom(_, message)        => Some(message)
+        case Custom(_)                 => Some("custom message") //TODO?
```

Comment:
Maybe we could add additional optional property - description which will be used for that purpose?